app/Eloquent/Reputation.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace App\Eloquent;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Reputation extends Model
+{
+    protected $fillable = [
+    	'point',
+    	'user_id',
+    	'target_type',
+    	'target_id',
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Generic.WhiteSpace.DisallowTabIndent.TabsUsed</br>**Severity**: error</br>**File**: app/Eloquent/Reputation.php</br>Spaces must be used to indent lines; tabs are not allowed